src/index.html

```diff
@@ -13,5 +13,140 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="inputs">
+          <label for="surname">Surname:</label>
+          <input
+          type="text"
+          name="surname"
+          id="surname"
+          required
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            id="surname"
            required
            autocomplete="off"
          >
```
Відступи і форматування. Можна почитати тут
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation

Варто скрізь пофіксити у коді

Reply:
Вроді все виправив
Показує помилку що немає placeholder хоча він є